bigquery/bq_funcs.js

```diff
@@ -0,0 +1,3 @@
+function transform(line) {
```

Comment:
What's this for?

Reply:
For GCloud Dataflow you need to have a function on the ingestion, it is used to transform the data, in our case it's already in the right format so we just return it.

Reply:
Please include this in the documentation. Also explicitly how to set things up and connect it.